qualtran/bloqs/rotations/programmable_rotation_gate_array.ipynb

```diff
@@ -0,0 +1,145 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "cd9af9c1",
+   "metadata": {
+    "cq.autogen": "title_cell"
+   },
+   "source": [
+    "# Programmable Rotation Gate Array"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d1abb757",
+   "metadata": {
+    "cq.autogen": "top_imports"
+   },
+   "outputs": [],
+   "source": [
+    "from qualtran import Bloq, CompositeBloq, BloqBuilder, Signature, Register\n",
+    "from qualtran import QBit, QInt, QUInt, QAny\n",
+    "from qualtran.drawing import show_bloq, show_call_graph, show_counts_sigma\n",
+    "from typing import *\n",
+    "import numpy as np\n",
+    "import sympy\n",
+    "import cirq"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c850551e",
+   "metadata": {
+    "cq.autogen": "ProgrammableRotationGateArray.bloq_doc.md"
+   },
+   "source": [
+    "## `ProgrammableRotationGateArray`\n",
+    "An implementation of `ProgrammableRotationGateArrayBase` base class\n",
+    "\n",
+    "\n",
+    "This implementation of the `ProgrammableRotationGateArray` base class expects\n",
+    "all interleaved_unitaries to act on the `rotations_target` register.\n",
+    "\n",
+    "See docstring of `ProgrammableRotationGateArrayBase` for more details."
```

Comment:
it looks like all the actual documentation is in the base class. @tanujkhattar how do you want to document this? We can document the base class with an empty list of bloq examples; but I think we still need a more helpful docstring for the concrete implementation rather than its by-line being "an implementation of the base class"